WebRoot/js/zimbraMail/abook/view/ZmContactSplitView.js

```diff
@@ -1133,7 +1133,8 @@ ZmContactSimpleView.prototype._modifyContact =
 function(ev) {
 	ZmContactsBaseView.prototype._modifyContact.call(this, ev);
 
-	if (ev.getDetail("fileAsChanged")) {
+	//TODO: `contactImageChanged` event should be handled separately and should just update current row instead of relayout
```

Comment:
can we file a performance improvement ticket for this, just to make sure we don't forget this in future